File: python/resistor-color-duo/resistor_color_duo.py
"""Double color resistor calculator"""

COLORS = {"black": 0,
          "brown": 1,
          "red": 2,
          "orange": 3,
          "yellow": 4,
          "green": 5,
          "blue": 6,
          "violet": 7,
          "grey": 8,
          "white": 9}


def value(colors):
    val_str = f'{COLORS[colors[0]]}{COLORS[colors[1]]}'
    return int(val_str)

File: solutions/csharp/resistor-color-duo/1/ResistorColorDuo.cs
using System;
using System.Collections.Generic;

public static class ResistorColorDuo
{
    private static readonly Dictionary<string, int> _colorValues = new()
    {
        ["black"] = 0,
        ["brown"] = 1,
        ["red"] = 2,
        ["orange"] = 3,
        ["yellow"] = 4,
        ["green"] = 5,
        ["blue"] = 6,
        ["violet"] = 7,
        ["grey"] = 8,
        ["white"] = 9
    };

    public static int Value(string[] colors) => int.Parse($"{_colorValues[colors[0]]}{_colorValues[colors[1]]}");
}
